pages/Configuring/Dispatchers.md

```diff
@@ -97,8 +97,6 @@ You have eight choices:
 
 - Relative workspace on monitor: e.g. `m+1`, `m-1` or `m+3`
 
-- Relative workspace on monitor including empty workspaces: e.g. `r+1` or `r-3`
```

Comment:
Why remove this? I don't see how it's related to this PR.

Reply:
No clue how that got in here. Good catch.